app/server/ruby/lib/sonicpi/synths/synthinfo.rb

```diff
@@ -118,7 +118,7 @@ def arg_doc(arg_name)
         info = arg_info[arg_name.to_sym]
         info[:doc] if info
       end
-
+i. e.
```

Comment:
typo?

Reply:
I was wondering where my »i. e.« went when I had typed it into the search box and it didn't show up there. :) Thanks for spotting it.